import { DomainError } from './domain.error';

export class NotFoundError extends DomainError {
  constructor(message = 'Recurso n√£o encontrado.') {
    super(message, 404);
  }
}
